packages/components/src/Chips/ChipTypes.tsx

```diff
@@ -0,0 +1,54 @@
+import { ReactElement } from "react";
+import { ChipButtonProps } from "./InternalChipButton";
+import { AvatarProps } from "../Avatar";
+import { IconProps } from "../Icon";
+
+export interface InternalChipProps {
+  /**
+   * Label of the chip.
+   */
+  readonly label: string;
+
+  /**
+   * Throws a console warning when the chip label goes over 24 characters.
+   */
+  readonly warnOnLongLabels?: boolean;
```

Comment:
This is a strange prop to me. I feel this should just be built in if we want to do this.

Reply:
yah, thing is we only want it on single/multi select as the usage for that is much more strict. The dismissible ones on the other hand doesn't need it. I can move it to the single/multi component tho. What do you think?

Reply:
removed it! for now... sorry chris

Reply:
Fair 😭  @eddysims the intention of these is actually to warn devs and encourage them to talk with their designers, but I think if we want to do something like this it could be a separate pattern we figure out outside of this PR!